README.md

```diff
@@ -1,46 +1,67 @@
 
 ## Bidding and Auction services
 
 * [Bidding and Auction services](bidding_auction_services_api.md)
+* [Bidding and Auction services: Protected App Signals Ad Selection](bidding_auction_services_protected_app_signals.md)
 * [Bidding and Auction services onboarding and self-serve guide](bidding_auction_services_onboarding_self_serve_guide.md)
 * [Bidding and Auction services payload optimization](https://github.com/privacysandbox/fledge-docs/blob/main/bidding-auction-services-payload-optimization.md)
 * [Bidding and Auction services system design](bidding_auction_services_system_design.md)
 * [Bidding and Auction services multi seller auctions](https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_multi_seller_auctions.md)
-* [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
-* [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 * [Protected Audience event level reporting with Bidding and Auction services](bidding_auction_event_level_reporting.md)
+* Cloud support
+    * AWS: [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
+    * GCP: [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 
 ## Key/Value service
 
 * [Key/Value service trust model](key_value_service_trust_model.md)
-* [Key/Value service user-defined functions (UDFs)](key_value_user_defined_functions.md)
+* [Key/Value service use cases](key_value_service_use_cases.md)
+* [Key/Value service user-defined functions (UDFs)](key_value_service_user_defined_functions.md)
 * [Key/Value sharding](key_value_sharding.md)
 
+## Ad Retrieval service
+
+* [Protected App Signals: Ad Retrieval service](https://github.com/privacysandbox/fledge-key-value-service/blob/main/docs/ad_retrieval_overview.md)
+
+## Inference
+
+* [Bidding and Auction services: Inference Overview](inference_overview.md)
+
 ## Server productionization
 
 * [Production operation](production_operation.md)
 * [Debugging protected audience api services](debugging_protected_audience_api_services.md)
 * [Monitoring protected audience api services](monitoring_protected_audience_api_services.md)
 
+## Server cost
+
+* [Bidding and Auction services: Cost](bidding_auction_cost.md)
+
 ## Client and server integration
 
+### Protected Audience
 * [Browser - Bidding and Auction services integration](https://github.com/WICG/turtledove/blob/main/FLEDGE_browser_bidding_and_auction_API.md)
 * [Android - Bidding and Auction services integration high level document](https://developer.android.com/design-for-safety/privacy-sandbox/protected-audience-bidding-and-auction-services)
 
+### Protected App Signals
+* [Android: Protected App Signals]([Protected App Signals](https://developer.android.com/design-for-safety/privacy-sandbox/protected-app-signals)
```

Comment:
1. This doesn't render well. 
2. Can use hyphen after Android to be consistent with PA. Following would be better:

[Android - Protected App Signals](https://developer.android.com/design-for-safety/privacy-sandbox/protected-app-signals)

Reply:
Fixed, thanks.